File: 240206/007/007.py
class Secret:
    def __init__(self, code, place, time):
        self.code = code
        self.place = place
        self.time = time

c, p, t = input().split()

answer = Secret(c, p, t)

print('secret code : ' + answer.code)
print('meeting point : ' + answer.place)
print('time : ' + answer.time)

File: 240812/007/007.js
const fs = require("fs");
let [code, place, time] = fs.readFileSync(0).toString().trim().split(" ");

class Secret {
    constructor(code, place, time) {
        this.code = code;
        this.place = place;
        this.time = time;
    }
}

const secret1 = new Secret(code, place, time);

console.log(`secret code : ${secret1.code}`);
console.log(`meeting point : ${secret1.place}`);
console.log(`time : ${secret1.time}`);